README.md

```diff
@@ -125,25 +125,31 @@ https://github.com/reactstrap/code-sandbox-examples
 Install dependencies:
 
 ```sh
-npm install
+yarn install
 ```
 
 Run examples at [http://localhost:8080/](http://localhost:8080/) with webpack dev server:
 
 ```sh
-npm start
+yarn start
+```
+
+Run tests:
+
+```sh
+yarn test
 ```
 
-Run tests & coverage report:
+Get test coverage report:
 
 ```sh
-npm test
+yarn cover
```

Comment:
`yarn cover` actually runs tests and gets coverage report. So I think we can just run `yarn cover` to Run tests and coverage report.

Reply:
Good call. I pushed another commit using the former language with `yarn cover`.